spec/item_repository_spec.rb

```diff
@@ -26,7 +26,7 @@
 
   describe '#all' do
     it 'returns a list of all items' do
-      allow_any_instance_of(ItemRepository).to receive(:create_items).and_return(MockData.items_as_hash)
+      allow_any_instance_of(ItemRepository).to receive(:create_items).and_return(MockData.items_as_hashes)
```

Comment:
Looks like you've gone through and changed all the places this is referenced. If all the tests pass this should be good to go.

Reply:
Yeah, all tests in the project are passing fully green 🟢